src/components/GlobalPreferences/CustomLabels/ShareModal.js

```diff
@@ -20,91 +20,75 @@ function ShareModal({ inputRef, onClose, onCopy, onFocus, link, visible }) {
 
   return (
     <Modal visible={visible} onClose={onClose} zIndex={10001}>
-      <header
+      <h1
         css={`
-          font-size: 22px;
-          line-height: 38px;
-          font-weight: bold;
+          ${textStyle('title2')}
         `}
       >
         Share custom labels
-      </header>
-      <main style={{ marginTop: `${2 * GU}px` }}>
+      </h1>
+      <main
+        css={`
+          margin-top: ${2 * GU}px;
+        `}
+      >
         <div
           css={`
-            font-size: 15px;
-            line-height: 22px;
+            ${textStyle('body2')}
+            margin-bottom: ${2.5 * GU}px;
           `}
         >
           These labels will be shared with everyone that has access to this
           link.
         </div>
-        <div style={{ marginTop: `${2.5 * GU}px` }}>
+        <div>
           <div
             css={`
               font-size: 12px;
               line-height: 16px;
               text-transform: uppercase;
+              margin-bottom: ${1 * GU}px;
             `}
           >
             Link
           </div>
-          <div
+          <TextInput
```

Comment:
I wonder if it would be useful to provide a component for these, like `TextCopy`?

Reply:
Hmm yes, I think this is going to be a fairly common use case in a lot of apps!

Reply:
Moved to https://github.com/aragon/aragon-ui/issues/504.